src/Tickets/Commerce/Gateways/Stripe/REST/Webhook_Endpoint.php

```diff
@@ -0,0 +1,191 @@
+<?php
+
+namespace TEC\Tickets\Commerce\Gateways\Stripe\REST;
+
+use TEC\Tickets\Commerce\Gateways\Contracts\Abstract_REST_Endpoint;
+use TEC\Tickets\Commerce\Gateways\Stripe\Settings;
+
+use TEC\Tickets\Commerce\Gateways\Stripe\Status;
+use TEC\Tickets\Commerce\Gateways\Stripe\Webhooks;
+use WP_REST_Server;
+use WP_REST_Request;
+use WP_REST_Response;
+
+use Tribe__Utils__Array as Arr;
+
+/**
+ * Class Webhook Endpoint.
+ *
+ * @since   TBD
+ *
+ * @package TEC\Tickets\Commerce\Gateways\Stripe\REST
+ */
+class Webhook_Endpoint extends Abstract_REST_Endpoint {
+
+	const WATCHED_EVENTS = [
```

Comment:
Do we want to add a getter with these values filtered? This could prevent us from hotfixing in case we missed some, and users to add other watched events in case they want (maybe for particular payment methods they have to watch other events).